generators/server/templates/src/main/java/package/config/_WebConfigurer.java

```diff
@@ -116,11 +116,14 @@ public void customize(ConfigurableEmbeddedServletContainer container) {
         // CloudFoundry issue, see https://github.com/cloudfoundry/gorouter/issues/64
         mappings.add("json", "text/html;charset=utf-8");
         container.setMimeMappings(mappings);<% if (!skipClient) { %>
-        // Set document root
-        if (env.acceptsProfiles(Constants.SPRING_PROFILE_PRODUCTION)) {
-            container.setDocumentRoot(new File("<%= CLIENT_DIST_DIR %>"));
-        } else if (env.acceptsProfiles(Constants.SPRING_PROFILE_DEVELOPMENT)) {
-            container.setDocumentRoot(new File("<%= CLIENT_MAIN_SRC_DIR %>"));
+
+        // Set document root if we're not running from a jar/war
+        if (getClass().getProtectionDomain().getCodeSource().getLocation().getProtocol().equals("file")) {
```

Comment:
I think it would be better that the String `"file"` is on the left of the .equals to be complient with the squid rule [S1132 : Strings literals should be placed on the left side when checking for equality](https://dev.eclipse.org/sonar/rules/show/squid:S1132)